bin/docs.py

```diff
@@ -34,20 +40,26 @@
     names = lookup.keys()
     names.sort()
 
+    #now write the license out
+    _license = io.open(license, "w")
+    _license.write("# License\n\n")
+
     now = datetime.datetime.now()
     ymd = now.strftime("%Y-%m-%d")
 
     docs = io.open(readme, "w")
     docs.write("# sources\n\n")
 
+    docs.write("Crediting Who's On First is recommended and linking back to the License is required. For example:\n\n> Data from Who's On First. [License](https://github.com/whosonfirst/whosonfirst-sources/blob/master/LICENSE.md).\n\nThe Who's On First dataset is both original work and a modification of existing open data (sources listed below). Some of those open data projects do require attribution. We have listed all sources below in the full [license file](https://github.com/whosonfirst/whosonfirst-sources/blob/master/LICENSE.md).\n\n")
```

Comment:
Update the URLs to be true: 

- https://whosonfirst.org/docs/licenses/

(Adjust for final resting place)

Reply:
(2x)

Reply:
fixed 09a748c and 2f31015